aplt/client.py

```diff
@@ -146,7 +146,7 @@ def expect_notification(self, command):
     def wait(self, command):
         """Wait for a period of time"""
         self._waiting = True
-        self.setTimeout(command.time)
+        self.setTimeout(float(command.time))
```

Comment:
Isn't this supposed to be an int?
https://twistedmatrix.com/documents/8.2.0/api/twisted.protocols.policies.TimeoutMixin.html#setTimeout